### ISSUE TITLE
Lexical error when encountering UTF-8 in .bib file

### ISSUE THREAD

**Issue by mihaeu** [michel-kraemer/citeproc-java#31]
Hey there,

love your project! Automate all the things!

I got the following exception when calling citeproc-java on a `.bib` file containing a unicode character (namely “):

```
Exception in thread "main" org.jbibtex.TokenMgrError: Lexical error at line 1, column 248.  Encountered: "\u201c" (8220), after : ""
	at org.jbibtex.LaTeXParserTokenManager.getNextToken(LaTeXParserTokenManager.java:303)
	at org.jbibtex.LaTeXParser.jj_scan_token(LaTeXParser.java:408)
	at org.jbibtex.LaTeXParser.jj_3R_6(LaTeXParser.java:270)
	at org.jbibtex.LaTeXParser.jj_3_3(LaTeXParser.java:253)
	at org.jbibtex.LaTeXParser.jj_2_3(LaTeXParser.java:242)
	at org.jbibtex.LaTeXParser.String(LaTeXParser.java:190)
	at org.jbibtex.LaTeXParser.Object(LaTeXParser.java:66)
	at org.jbibtex.LaTeXParser.ObjectList(LaTeXParser.java:46)
	at org.jbibtex.LaTeXParser.LaTeX(LaTeXParser.java:21)
	at org.jbibtex.LaTeXParser.parse(LaTeXParser.java:16)
	at de.undercouch.citeproc.bibtex.BibTeXConverter.toItemData(BibTeXConverter.java:161)
	at de.undercouch.citeproc.bibtex.BibTeXConverter.toItemData(BibTeXConverter.java:143)
	at de.undercouch.citeproc.bibtex.BibTeXItemDataProvider.addDatabase(BibTeXItemDataProvider.java:32)
	at de.undercouch.citeproc.CSLTool.readBibliographyFile(CSLTool.java:325)
	at de.undercouch.citeproc.CSLTool.run(CSLTool.java:255)
	at de.undercouch.citeproc.CSLTool.main(CSLTool.java:139)
```

This is the `.bib` file in question: [ref.txt](https://github.com/michel-kraemer/citeproc-java/files/594792/ref.txt)

**Comment by michel-kraemer** [comment]
Thanks for reporting. I cannot reproduce the issue with the latest version in master.

I suppose you're still using the latest release 0.6? Version 1.0 will be released quite soon. I just need to update the docs quickly (hopefully tomorrow) and find a solution for #28.

Do you need the new version urgently?

**Comment by mihaeu** [comment]
I just tried to build the `develop` branch using `./gradlew jar` on Fedora 23, but without success. So couldn't check if that is the problem or not.

Could this be linked to #28 as well, because I'm also using Java 8.

And no I do not need the new version urgently.

**Comment by michel-kraemer** [comment]
What error message do you get?

**Comment by mihaeu** [comment]
The gradle build finished, but the manifest for the main jar is missing and the main class for the tool jar:

```
$> ./gradlew jar

$> java -jar citeproc-java-tool/build/libs/citeproc-java-tool-1.0.0-SNAPSHOT.jar
Error: Could not find or load main class de.undercouch.citeproc.CSLTool

$> java -jar citeproc-java/build/libs/citeproc-java-1.0.0-SNAPSHOT.jar
no main manifest attribute, in citeproc-java/build/libs/citeproc-java-1.0.0-SNAPSHOT.jar
```

**Comment by michel-kraemer** [comment]
If you want to build the command line tool use `./gradlew installDist`. You'll find the tool under `citeproc-java-tool/build/install`. The directory will contain a shell script (or batch file).

I fixed #28 in the meantime. I'll try to release the new version today or tomorrow.

Cheers,
Michel

**Comment by mihaeu** [comment]
That did the trick! Thanks. Everything working fine on `develop`

Thanks so much!!!

**Comment by michel-kraemer** [comment]
I just published version 1.0.0 by the way!

Cheers,
Michel
